browser/search_engines/search_engine_tracker.cc

```diff
@@ -239,6 +244,16 @@ void SearchEngineTracker::RecordWebDiscoveryEnabledP3A() {
       kWebDiscoveryAndAdsMetric,
       enabled && profile_prefs_->GetBoolean(
                      brave_ads::prefs::kOptedInToNotificationAds));
+
+  // Record web discovery default engine metric
+  if (enabled) {
+    UMA_HISTOGRAM_ENUMERATION(kWebDiscoveryDefaultEngineMetric,
+                              current_default_engine_);
+  } else {
+    UMA_HISTOGRAM_EXACT_LINEAR(
```

Comment:
why this one is linear, and the one above is enumeration? Anyway, it is recommended to use functions instead of macros if you record same histogram in multiple spots